### ISSUE TITLE
Align function definition equals sign

### ISSUE THREAD

**Issue by Rhywun** [ennocramer/floskell#27]
Hi, is there a way to align the equals signs here, such that this:

```
sorted :: [Integer] -> Bool
sorted [] = True
sorted [_] = True
sorted (x:y:zs) = x < y && sorted (y : zs)
```

becomes this:

```
sorted :: [Integer] -> Bool
sorted []       = True
sorted [_]      = True
sorted (x:y:zs) = x < y && sorted (y : zs)
```

Thanks!

**Comment by ennocramer** [comment]
This is currently not possible, though it should not be hard to add. All the necessary pieces are already in place.

**Comment by Rhywun** [comment]
I would appreciate it, much thanks!

**Comment by ennocramer** [comment]
And there you go.  Implemented in `master`/`ec172d9d`. Cheers!
